### ISSUE TITLE
Mypal and otehr OS

### ISSUE THREAD

**Issue by ROBERT-MCDOWELL** [Feodor2/Mypal#328]
Hello, is mypal can work on other OS like win10, mac OSX or linux?
I would like to offer a browser alternative for my users who need Flash environment as I need time to convert my whole platform in HTML5/JS which is a big challenge, thanks

**Comment by Feodor2** [comment]
Win10 may be, but i don't use it so don't check, but i am not going to make mypal for linux, osx is such a rarity for me and never supposed to do something, by the way i heard new osx version more horrible spyware then win10 so no thanks.

Take original palemoon it covers for all your named os.

**Comment by ROBERT-MCDOWELL** [comment]
crazy time of big brother indeed, that's why I still use xp... I asked it for users on a project. centaury works great on xp though.

**Comment by ROBERT-MCDOWELL** [comment]
@Feodor2
if interested to provide last good Flash plugin I have a distribution license so maybe you can provide mypal + centaury with the last NPAPI version without the time bomb (32.0.0.344)

**Comment by Feodor2** [comment]
Sure, i can do a page about this?

**Comment by ROBERT-MCDOWELL** [comment]
yes, or just incorporate the plugin directly into your installers? the most easy for users, for now I would like to redirect the usual "download flash player" link to centaury or mypal, at least for windows.

**Comment by AroKol78** [comment]
(last NPAPI version without the time bomb 32.0.0.371)
> plugin directly into your installers

I don't know if everyone wants to

I don't need it - I don't use it

flash player add to http://myapl.eu3.biz/exts/ (in my opinion)

**Comment by ROBERT-MCDOWELL** [comment]
ok, for centaury too? Direct download link from a github repo?

**Comment by AroKol78** [comment]
"for centaury too?" yes
everyone can choose what they want
https://archive.org/download/flashplayerarchive/pub/flashplayer/installers/archive/

32.0.0.371 for different systems 
https://ia803201.us.archive.org/view_archive.php?archive=/18/items/flashplayerarchive/pub/flashplayer/installers/archive/fp_32.0.0.371_archive.zip
(This is not file zip - normal page)

**Comment by ROBERT-MCDOWELL** [comment]
good catch! I found only 32.0.0.344 on archive.org before... my mistak to not keep last one year versions in my backup....
there are plenty of games and companies still using flash for games, remote control, inudstrial appliance etc...

**Comment by ROBERT-MCDOWELL** [comment]
uploaded to a git repo https://github.com/ROBERT-MCDOWELL/flash-plugins/

**Comment by ROBERT-MCDOWELL** [comment]
wooww, I just found that flash player continues to be updated in China
flash.cn, version 34 for now... intriguing...

**Comment by AroKol78** [comment]
> updated in China flash.cn, version 34 for now... intriguing...

added/contains malware
